### ISSUE TITLE
Support for several build config (Grunt style)

### ISSUE THREAD

**Issue by felicienfrancois** [atom/grunt-electron-installer#20]
The following gruntfile should be supported

```javascript
...
"create-windows-installer": {
    "myfirstcfg" : {
        appDirectory: "some/path",
        outputDirectory: "target/path"
    },
   "anothercfg": {
        appDirectory: "another/path",
        outputDirectory: "target/other"
    }
},
...
```

But I get the following error when i run `grunt create-windows-installer:myfirstcfg`:
```
Required config property "create-windows-installer.appDirectory" missing
```

**Comment by techninja** [comment]
Seconded. April 10?! Sheesh. I'll see if I can't make a PR for this.

**Comment by techninja** [comment]
Done! #43
I even tested this and it looks like I'll be running it with my builds as well :)

**Comment by techninja** [comment]
Woot! This just got merged into 0.36.0! @felicienfrancois Test this out and let us know if this should be closed.
